components/invitation_modal/invitation_modal_members_step/__snapshots__/invitation_modal_members_step.test.jsx.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/invitation_modal/InvitationModalMembersStep should match the snapshot 1`] = `
+<ContextConsumer>
```

Comment:
Usually this isn't a good sign when a snapshot goes from 279 lines to 13.
This is probably indicative of a bad test setup.
Looking down it looks like you're importing the connected redux component to the test. You'll want to import just the raw unconnected component to make sure the snapshots look right.

Reply:
Cool - imported the component directly and updated the snapshot, which has it now back to its master state.